queries/cpon/highlights.scm

```diff
@@ -15,32 +15,40 @@
   "d" @character.special
   (_) @string.special)
 
-(_ key: (_) @label)
+(_
+  key: (_) @property)
 
 (number) @number
 
-(float) @float
+(float) @number.float
 
 (boolean) @boolean
 
 (null) @constant.builtin
 
 ; Punctuation
-
 [
   ","
   ":"
 ] @punctuation.delimiter
 
-[ "{" "}" ] @punctuation.bracket
+[
+  "{"
+  "}"
+] @punctuation.bracket
 
-[ "[" "]" ] @punctuation.bracket
+[
+  "["
+  "]"
+] @punctuation.bracket
 
-[ "<" ">" ] @punctuation.bracket
+[
+  "<"
+  ">"
+] @punctuation.bracket
 
-(("\"" @conceal)
- (#set! conceal ""))
+(("\"" @string)
```

Comment:
```suggestion
(("\"" @conceal)
```

Reply:
Please only comment on **backporting** mistakes. 

If there are changes you'd like to make on `main`, just open a PR and I'll merge and cherry-pick the commits for this PR.

Reply:
Or wait until this is merged and fix it on `master` -- the less I have to rebase, the better :)